pkgs/top-level/all-packages.nix

```diff
@@ -11943,7 +11943,7 @@ in
 
   amsn = callPackage ../applications/networking/instant-messengers/amsn { };
 
-  antimony = qt5.callPackage ../applications/graphics/antimony {};
+  antimony = qt55.callPackage ../applications/graphics/antimony {};
```

Comment:
This bug depends on what you do have in the system path. For example, I have 5.6 libraries there, so it's the opposite for me. In this situation let's use default Qt (5.6) instead -- more chances it would work for a given person.